grid_map_filters/src/grid_map_filters/ColorFillFilter.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * ColorFillFilter.cpp
+ *
+ *  Created on: Sep 14, 2017
+ *      Author: Peter Fankhauser
+ *   Institute: ETH Zurich, ANYbotics
+ */
+
+#include <grid_map_filters/ColorFillFilter.hpp>
+
+#include <grid_map_core/grid_map_core.hpp>
+#include <pluginlib/class_list_macros.hpp>
+
+#include <Eigen/Dense>
+
+#include <string>
+
+namespace grid_map
+{
+
+ColorFillFilter::ColorFillFilter()
+: r_(0.0),
+  g_(0.0),
+  b_(0.0)
+{
+}
+
+ColorFillFilter::~ColorFillFilter()
+{
+}
+
+bool ColorFillFilter::configure()
+{
+  if (!getParam(std::string("red"), r_)) {
+    RCLCPP_ERROR(
+      logging_interface_->get_logger(), "Color fill filter did not find parameter `red`.");
+    return false;
+  }
+  RCLCPP_DEBUG(logging_interface_->get_logger(), "Color fill filter red is = %f.", r_);
+
+  if (!getParam(std::string("green"), g_)) {
+    RCLCPP_ERROR(
+      logging_interface_->get_logger(),
+      "Color fill filter did not find parameter `green`.");
+    return false;
+  }
+  RCLCPP_DEBUG(logging_interface_->get_logger(), "Color fill filter green is = %f.", g_);
+
+  if (!getParam(std::string("blue"), b_)) {
+    RCLCPP_ERROR(
+      logging_interface_->get_logger(), "Color fill filter did not find parameter `blue`.");
+    return false;
+  }
+  RCLCPP_DEBUG(logging_interface_->get_logger(), "Color fill filter blue is = %f.", b_);
+
+  // if (!getParam(std::string("mask_layer"), maskLayer_)) {}
```

Comment:
That looks fishy.

Reply:
I commented it by mistake. Will bring it back without the condition, because the linter complains about empty condition bodies.